ONSdigital/eq-author-api
# Issues developing against node 10

### What is the context of this PR?
Have run `yarn upgrade` this fixes:
1. Build errors on mac with `10.12.0` related to `fsevents`
1. Inability to run test watch with `10.12.0` on mac.
1. `docker-compose` `nodemon` file watching does not work.

Also have added new `nodemon` flag `-L` to get it to pick up file changes.

### How to review
1. Ensure local node version is `10.12.0`
1. Prove on your local that file watching (with `yarn test --watch` and `docker-compose`) don't work on `master` after nuking `node_modules` and running `yarn`.
1. Switch to this branch - run `yarn`
1. Ensure there are no errors during `yarn`
1. Ensure you can run tests and app locally.
